### PR TITLE
Refactor and fix debugger integration tests for new project API 

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
$subject.
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/26909

## Remarks
However, few of the test cases related to ballerina test debugging are still disabled until the test support is fixed with the new project API. Please refer https://github.com/ballerina-platform/ballerina-lang/issues/26942.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [x] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [x] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/modules/myService/helloService.bal [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//import ballerina/http;
```
Comment (azinneera) [inline tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/modules/myService/helloService.bal:1]: Do we need this completely commented out file?

Comment (NipunaRanasinghe) [reply tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/modules/myService/helloService.bal]: The project won't compile as this module is having std-lib dependencies, so the plan is to migrate the before mentioned HTTP service based tests into the distribution repo. +1 to remove the related content until that. Fixed with 9e0ad3a

File: tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/foo/tests/main_test.bal [hunk 2]
```diff
@@ -1,65 +0,0 @@
-// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (praveennadarajah) [inline tests/jballerina-debugger-integration-test/src/test/resources/project-based-tests/breakpoint-tests/src/foo/tests/main_test.bal:1]: Any reason to remove this file completely? This file is used in `MultiModuleTestDebugTest` and it is currently disabled now.

File: tests/jballerina-debugger-integration-test/src/test/resources/testng.xml [hunk 3]
```diff
@@ -30,21 +30,30 @@ under the License.
...
+            <class name="org.ballerinalang.debugger.test.remote.BallerinaRunRemoteDebugTest"/>
+            <class name="org.ballerinalang.debugger.test.remote.BallerinaTestRemoteDebugTest"/>
+            <class name="org.ballerinalang.debugger.test.remote.BallerinaBuildRemoteDebugTest"/>
+
+            <!--Debug engaging tests for ballerina commands -->
+            <class name="org.ballerinalang.debugger.test.adapter.run.SingleBalFileRunDebugTest"/>
+            <class name="org.ballerinalang.debugger.test.adapter.ModuleBreakpointTest"/>
+            <class name="org.ballerinalang.debugger.test.adapter.run.MultiModuleRunDebugTest"/>
+            <!--<class name="org.ballerinalang.debugger.test.adapter.test.MultiModuleTestDebugTest"/>-->
+            <!--<class name="org.ballerinalang.debugger.test.adapter.build.MultiModuleBuildDebugTest"/>-->
```
Comment (praveennadarajah) [inline tests/jballerina-debugger-integration-test/src/test/resources/testng.xml:43]: Shall we enable these tests as well? Do we have any issues in that?

File: tests/jballerina-debugger-integration-test/src/test/resources/testng.xml [hunk 4]
```diff
@@ -30,21 +30,30 @@ under the License.
...
+
+            <!--Breakpoint tests-->
+            <class name="org.ballerinalang.debugger.test.adapter.ControlFlowDebugTest"/>
+            <class name="org.ballerinalang.debugger.test.adapter.CallStackDebugTest"/>
+            <!--<class name="org.ballerinalang.debugger.test.adapter.ServiceDebugTest"/>-->
+            <!--<class name="org.ballerinalang.debugger.test.adapter.LanguageConstructDebugTest"/>-->
```
Comment (praveennadarajah) [inline tests/jballerina-debugger-integration-test/src/test/resources/testng.xml:49]: Same [here](https://github.com/ballerina-platform/ballerina-lang/pull/26943/files#r523955227) as well.

Comment (praveennadarajah) [inline tests/jballerina-debugger-integration-test/src/test/resources/testng.xml:49]: Same [here](https://github.com/ballerina-platform/ballerina-lang/pull/26943/files#r523955227) as well.

File: tests/jballerina-debugger-integration-test/src/test/resources/testng.xml (hunk 3 above)
Comment (NipunaRanasinghe) [reply tests/jballerina-debugger-integration-test/src/test/resources/testng.xml:43]: Please refer #26942.

File: tests/jballerina-debugger-integration-test/src/test/resources/testng.xml (hunk 4 above)
Comment (NipunaRanasinghe) [reply tests/jballerina-debugger-integration-test/src/test/resources/testng.xml:49]: Please refer #26942.

**Review by praveennadarajah — APPROVED** [review]

**Pull request merged**